docs/components/Form/Form.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta } from "@storybook/addon-docs";
+import { Form } from "@jobber/components/Form";
+
+<Meta title="Components/Forms and Inputs/Form/Docs" component={Form} />
+
+# Form
+
+The Form component is a wrapper component that handles the submission and
+validation of forms.
+
+For more information about `validations` using any of the `Input` components,
+see the
+[InputText](../?path=/docs/components-forms-and-inputs-inputtext--input-text/#validation-message)
+documentation.
+
+## Design & usages guidelines
+
+The `<Form>` component has a lot of built-in features which rely on its internal
+state. To take advantage of these features, do not bypass the Form's internal
+state — the fields or inputs within the Form must have a `name` prop and **NOT**
+have a `value` and `onChange` prop.
+
+### Save button behaviour (Mobile)
+
+#### iOS
+
+On iOS, the save button will be fixed to the bottom of the viewport until the
+keyboard is opened.
+
+Once the keyboard is open, the save button will be inline beneath the Form's
+inputs, unless the Form is so short that it does not scroll. In this case, the
+save button will remain fixed above the keyboard.
+
+This prevents the user from accidentally submitting the Form before they have
+completed entering all the relevant information for their work, especially since
+many data points in Jobber cannot be edited once saved.
+
+#### Android
+
+On Android, the save button will always be inline with the contents of the Form.
+
+## Content guidelines
+
+### Inputs
+
+`<Form>` can accept various inputs and selection elements such as (but not
+limited to) `InputText`, `Select`, `Switch`, `Checkbox`, and `Chips`. They
+should be placed inside Cards to indicate grouping when relevant, and groups of
+Cards can be spaced appropriately using `ContentSection`.
+
+### Save Button label (Mobile)
+
+The `saveButtonLabel` property defaults to "Save", but should be made more
+verbose to add context for the user. Use the format "Save {object}", such as
+"Save Job". This helps clarify to the user that tapping the Save Button is not
+saving the single input they are editing, but the entire object.
+
+### Form errors
```

Comment:
For this section - if this strictly related to mobile only then I think we can maybe also put it in `Platform considerations`.